pkgs/by-name/ff/ffmpeg-livepeer/package.nix

```
@@ -0,0 +1,35 @@
+{
+  ffmpeg_7-full,
+  lib,
+  fetchFromGitHub,
+  apple-sdk_11,
+}:
+
+let
+  version = "7.0.1";
```

Since you're building off an untagged branch, the convention would be to add a `-unstable-` and then the date in ISO 8601 format suffix to the base version.

```suggestion
  version = "7.0.1-unstable-2024-07-10";
```